### PR TITLE
circles-payment-api chiado

### PR DESCRIPTION
**Pull request by JacqueGM**

### CODE REVIEW

**Review by llunaCreixent — COMMENTED** [review]
Swagger UI not avilable: `https://api.payment.staging.circles.garden/docs`.

**Comment by JacqueGM** [comment]
yeah it's not cause @juanenrisley suggested to changed it to payment.circles.garden and I just did the change the change of domain and propagation will take a bit

**Comment by llunaCreixent** [comment]
Trying to test with 
```
curl -X 'POST' \
  'https://payment.staging.circles.garden/transactions' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{ "target": "0xBA93240FeC6d9F220F96a92D8B1Ad414070fFb3e", "data": "0x6a761202000000000000000000000000cfeb869f69431e42cdb54a4f4f105c19c080a6010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000014000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000004b7bc0f7300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004147eee603379de8bcfe2e4abdc56a8747a43569927305f4dbd5f1d8060bc19bef43c93ce90999fb3846674a7fc3170fbc79d96ea8859fa62f07d101f359d50d161b00000000000000000000000000000000000000000000000000000000000000"}'
```

and getting the error:
```
{"statusCode":500,"error":"Internal Server Error","message":"GelatoRelaySDK/sponsoredCall: Failed with error: Chain id [] is not supported"}
```

**Comment by llunaCreixent** [comment]
An I don't see any logs in the container :)

**Comment by juanenrisley** [comment]
The env variable `CHAIN_ID` is empty in the `env` configmap

File: helm/circles-infra-suite/values-staging.yaml [hunk 1]
```diff
@@ -15,10 +15,23 @@ apiService:
...
+
+apiPaymentService:
+  # Docker image tag
+  imageTag: v0.1.0-4-g3df0140
+
+  # Ingress
+  host: api.payment.staging.circles.garden
+
+  # chain id
+  chaindID:  10200
```
Comment (JacqueGM) [inline helm/circles-infra-suite/values-staging.yaml:34]: @juanenrisley  here is the chainid, specified per chain bc this is for chiado

File: helm/circles-infra-suite/values-staging.yaml
Comment (juanenrisley) [inline helm/circles-infra-suite/values-staging.yaml:1]: The `apiPaymentService.corsRegex` value is missing.

Comment (JacqueGM) [reply helm/circles-infra-suite/values-staging.yaml:1]: added

File: helm/circles-infra-suite/templates/configmap.yaml [hunk 2]
```diff
@@ -60,3 +60,7 @@ data:
   DJANGO_ALLOWED_HOSTS: .{{.Values.host}}
   DJANGO_SETTINGS_MODULE: config.settings.production
   RELAYER_REDIS_URL: redis://{{.Values.redisService.name}}:{{.Values.redisService.port}}/0
+
+  # Payment API Configuration
+  CHAIN_ID: '{{.Values.apiPaymentService.chaindID}}'
```
Comment (juanenrisley) [inline helm/circles-infra-suite/templates/configmap.yaml:65]: The `apiPaymentService.chaindID` value has already single quotes, even i do not if they are neessary. Here, single quotes are used again.

Comment (JacqueGM) [reply helm/circles-infra-suite/templates/configmap.yaml:65]: removed

**Review by juanenrisley — APPROVED** [review]
👍🏿 LGTM

**Pull request merged**